### PR TITLE
fix: npm semantic versioning

### PR DESCRIPTION
**Pull request by ProteanCoding**
This is to fix the issue elicited with `npm explain @dcl/js-runtime`:
```
@dcl/js-runtime@7.4.10 dev
node_modules/@dcl/asset-packs/node_modules/@dcl/js-runtime
  @dcl/js-runtime@"7.4.10" from @dcl/asset-packs@1.19.0
  node_modules/@dcl/asset-packs
    @dcl/asset-packs@"1.19.0" from @dcl/inspector@7.5.1
    node_modules/@dcl/inspector
      @dcl/inspector@"7.5.1" from @dcl/sdk-commands@7.5.1
      node_modules/@dcl/sdk-commands
        @dcl/sdk-commands@"7.5.1" from @dcl/sdk@7.5.1
        node_modules/@dcl/sdk
          dev @dcl/sdk@"7.5.1" from the root project
          @dcl/sdk@"latest" from @dcl/quests-client@1.0.3
          node_modules/@dcl/quests-client
            @dcl/quests-client@"^1.0.3" from @dcl/sdk-commands@7.4.10
            node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/sdk-commands
              @dcl/sdk-commands@"7.4.10" from @dcl/sdk@7.4.10
              node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
                @dcl/sdk@"7.4.10" from @dcl/asset-packs@1.16.0
                node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs
                  @dcl/asset-packs@"^1.14.0" from @dcl/inspector@7.4.10
                  node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
                    @dcl/inspector@"7.4.10" from @dcl/sdk-commands@7.4.10
                  @dcl/asset-packs@"1.16.0" from @dcl/inspector@7.4.15
                  node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
                    @dcl/inspector@"7.4.15" from @dcl/sdk-commands@7.4.15
                    node_modules/@dcl/asset-packs/node_modules/@dcl/sdk-commands
                      @dcl/sdk-commands@"7.4.15" from @dcl/sdk@7.4.15
                      node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
                        @dcl/sdk@"7.4.15" from @dcl/asset-packs@1.19.0
            @dcl/quests-client@"^1.0.3" from @dcl/sdk-commands@7.4.15
            node_modules/@dcl/asset-packs/node_modules/@dcl/sdk-commands
              @dcl/sdk-commands@"7.4.15" from @dcl/sdk@7.4.15
              node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
                @dcl/sdk@"7.4.15" from @dcl/asset-packs@1.19.0
            @dcl/quests-client@"^1.0.3" from @dcl/sdk-commands@7.5.1
  @dcl/js-runtime@"7.4.10" from @dcl/asset-packs@1.16.0
  node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs
    @dcl/asset-packs@"^1.14.0" from @dcl/inspector@7.4.10
    node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
      @dcl/inspector@"7.4.10" from @dcl/sdk-commands@7.4.10
    @dcl/asset-packs@"1.16.0" from @dcl/inspector@7.4.15
    node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
      @dcl/inspector@"7.4.15" from @dcl/sdk-commands@7.4.15
      node_modules/@dcl/asset-packs/node_modules/@dcl/sdk-commands
        @dcl/sdk-commands@"7.4.15" from @dcl/sdk@7.4.15
        node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
          @dcl/sdk@"7.4.15" from @dcl/asset-packs@1.19.0
  @dcl/js-runtime@"7.4.10" from @dcl/sdk@7.4.10
  node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
    @dcl/sdk@"7.4.10" from @dcl/asset-packs@1.16.0
    node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs
      @dcl/asset-packs@"^1.14.0" from @dcl/inspector@7.4.10
      node_modules/@dcl/asset-packs/node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
        @dcl/inspector@"7.4.10" from @dcl/sdk-commands@7.4.10
      @dcl/asset-packs@"1.16.0" from @dcl/inspector@7.4.15
      node_modules/@dcl/asset-packs/node_modules/@dcl/inspector
        @dcl/inspector@"7.4.15" from @dcl/sdk-commands@7.4.15
        node_modules/@dcl/asset-packs/node_modules/@dcl/sdk-commands
          @dcl/sdk-commands@"7.4.15" from @dcl/sdk@7.4.15
          node_modules/@dcl/asset-packs/node_modules/@dcl/sdk
            @dcl/sdk@"7.4.15" from @dcl/asset-packs@1.19.0
```

### CODE REVIEW

**Review by cyaiox — CHANGES_REQUESTED** [review]
Hey @ProteanCoding! thanks for your contribution! 

could you please install these versions using the flag `--save-exact` and push the new changes with the `package-lock.json`?

getting the main version of the files `package*.json`, execute: 

1. `npm ci`
2. `npm i --save-exact @dcl/sdk@7.4.15`
3. `npm i --save-exact @dcl/js-runtime@7.4.15`

**Comment by ProteanCoding** [comment]
@cyaiox, I am sorry, but what you are suggesting is (most likely) completely incorrect.

The source of the issue has been _exactly_ that the version of `@dcl/sdk` and `@dcl/js-runtime` were locked to their listed versions, preventing the proper versions from being summoned by the consuming libraries (and then requiring the use of `overrides` to get out of the mess).

These two libraries are now at `7.5.1`. What you are suggesting would just keep things rather broken, I fear.

**Comment by ProteanCoding** [comment]
I also just found out that the textures of the car and the ice in my scene finally load properly in Firefox if I add the following to the URL: `&DISABLE_ASSET_BUNDLES&DISABLE_WEARABLE_ASSET_BUNDLES`.
Before then, the ice would look like this:
Screenshot 2024-06-07 203358

**Comment by ProteanCoding** [comment]
I can commit the `package-lock.json` that now comes from `npm i`, but I'm wondering if an `npm update` shouldn't be run first 🤔

File: package.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
   },
   "dependencies": {
     "@dcl-sdk/utils": "^1.2.8",
-    "@dcl/js-runtime": "7.4.10",
-    "@dcl/sdk": "7.4.15",
+    "@dcl/js-runtime": "^7.4.15",
```
Comment (ProteanCoding) [inline package.json:54]: Bumping this to `^7.4.15`, as it is no longer certain that `@dcl/sdk@7.4.10` can be used here.

**Comment by cyaiox** [comment]
@ProteanCoding, you're right! sorry for suggesting that version.

You should execute the steps using the SDK version `7.5.1`.

The main idea of keeping the version fixed is to avoid conflicts between SDK versions. So, by uploading the package and package-lock.json, we assure that the versions defined are working and that we're not missing any lib installed locally during development.

This asset-packs library serves static file items, and the systems use the project's SDK installed version.

**Comment by cazala** [comment]
closing since this has been addressed here: https://github.com/decentraland/asset-packs/pull/118

**Pull request closed without merging**